ch6/problem_1.md

```diff
@@ -1,2 +1,15 @@
+1. I think python evaluates expression in normal order, because the test raise the `RecursionError: maximum recursion depth exceeded`. This only happens when the language evaluates eagerly, since `p()` are evaluated when it is defined, which cause the program to run forever and the maximum recursion depth exceeded.
```

Comment:
This describes "applicative" order, not "normal". Otherwise this explanation is correct. Python does use eager evaluation.